builder/README.md

```diff
@@ -1,13 +1,12 @@
 # Next.js builder for Cloudflare
 
-## Build your app
+## Configure your app
 
 - update the `next.config.mjs` as follows
 
   ```typescript
   /** @type {import('next').NextConfig} */
   const nextConfig = {
-    output: "standalone",
```

Comment:
is this change right?

Reply:
Yep, we use an env flag while building next, no need for that.
(The build output stays the same - the only exception is the addition of the `standalone` folder containing the traced files + a minial server.js)

Reply:
ah. ok. thanks

Reply:
I love that we don't need to change the config. great work everyone!